webapp/tests/karma/ts/modules/tasks/tasks.component.spec.ts

```diff
@@ -150,12 +176,11 @@ describe('TasksComponent', () => {
       { _id: '2', emission: { _id: 'e2', dueDate: pastDate.format('YYYY-MM-DD') }, owner: 'b' },
     ];
     rulesEngineService.fetchTaskDocsForAllContacts.resolves(taskDocs);
-
     await new Promise(resolve => {
       sinon.stub(TasksActions.prototype, 'setTasksList').callsFake(resolve);
       getComponent();
     });
-
+    clock.tick();
```

Comment:
Why was this added here?

Reply:
I think it's the equivalent of` tick() `when using  `clock = sinon.useFakeTimers(now.valueOf())` but you are right, it's not needed anymore, the test works also without it.